Assets/Scripts/Player/Player Database/PlayerData.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+namespace Examen.Player.PlayerDatabase
+{
+    [Serializable]
+    public class PlayerData
+    {
+        public string id;
+        public int exp;
```

Comment:
Do these need to be public? Could maybe also make private and then use properties for access.

Reply:
Same as above, so sadly no